include/nds/arm9/math.h

```diff
@@ -319,6 +319,14 @@ static inline uint32_t sqrt64(uint64_t a)
     return sqrt64_result();
 }
 
+
+/// 32 bit floating point square root
+//  @param valid 32 bit positive floating point value. NaN and Inf are not handled.
```

Comment:
The comments need to start with three slashes in all lines or Doxygen won't see them. Also, leave an empty line with only three slashes right below the description.

Also, the note about NaN and Inf is important, you should use `@warning` so that it appears very clearly in the doxygen documentation.